include/zephyr/dt-bindings/adc/nrf-saadc-haltium.h

```diff
@@ -0,0 +1,19 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ */
+
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_ADC_NRF_SAADC_NRF54H_H_
```

Comment:
nits: 2024 -> 2025 and NRF54H -> HALTIUM

Reply:
done